EAGLE Libraries/HyTechDevices.lbr

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE eagle SYSTEM "eagle.dtd">
-<eagle version="9.6.2">
+<eagle version="9.5.0">
```

Comment:
I think your eagle version is a little behind recent, as they went from 9.6.2 to 9.5.0.